fastlane/Fastfile

```diff
@@ -34,10 +36,17 @@ platform :android do
       if options[:build_type] == 'Debug'
         0
       else
-        google_play_track_version_codes(
-          package_name: package_name,
-          track: 'internal'
-        ).max
+        begin
+          google_play_track_version_codes(
+            package_name: package_name,
+            track: 'internal'
+          ).max
+        rescue Exception => e
+          unless e.message == "Google Api Error: Invalid request - Package not found: #{package_name}."
```

Comment:
question (non-blocking): Is this just temporary handling until the app is added?

Reply:
This required a temporary workaround when we created the staging app, it requires a temporary workaround now, and it will require a temporary workaround in the not inconceivable future where we want dev-silver. I don’t think it increases the maintenance burden to leave it in long-term.